packages/packages.csv

```diff
@@ -106,14 +107,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
 Microsoft.AspNetCore.StaticFiles,ship
 Microsoft.AspNetCore.TestHost,ship
 Microsoft.AspNetCore.Testing,noship
-Microsoft.AspNetCore.WebSockets.Client,ship
+Microsoft.AspNetCore.WebSockets.Client,noship
 Microsoft.AspNetCore.WebSockets.Protocol,ship
 Microsoft.AspNetCore.WebSockets.Server,ship
 Microsoft.AspNetCore.WebUtilities,ship
 Microsoft.Cci,ext
 Microsoft.CodeAnalysis.*,ext
 Microsoft.CSharp,ext
-Microsoft.Data.Sqlite,ship
+Microsoft.Data.Sqlite,ship-ext
```

Comment:
`ship-ext` is for external components that we ship. The effect is that we don't code-sign them, but we do publish them to NuGet.org.